tests/structuredheadings/keyboardshortcuts.js

```diff
@@ -17,7 +19,10 @@
 
     "H2 to H3 from Tab": function () {
       this.editorBot.setHtmlWithSelection("<h2>^Heading</h2>");
-      this.editor.fire("key", { keyCode: tabKey });
+      this.editor.editable().fire(
+        "keydown",
+        new CKEDITOR.dom.event({ keyCode: tabKey }) // eslint-disable-line new-cap
+      );
```

Comment:
before, we weren't using actual events, just objects with the correct keys in place.  This fails once you bubble events up to the original listeners, who use more fields from the event objects